### DISCUSSION TITLE
Switch to "Listener" (and back to initial instrument) with a shortcut

### DISCUSSION THREAD

**Discussion by Andy081549** [jamulussoftware/jamulus#3002] [Ideas]
Sometimes I just want to show the participants of a session that I am only listening and will not play over anybody. It would be great if there was a shortcut to changr the role to Listener quickly and back to the normal Instrument.

**Comment by ann0see** [comment]
Hi,

I have an unpopular approach, but JSON-RPC and a small script listening for a keyboard shortcut could do the trick.
